packages/components/CallToActionSection.tsx

```diff
@@ -1,8 +1,7 @@
 import { Button, Flex, Heading, Stack, Text } from '@chakra-ui/react';
 
 import { useModalActions } from '@packages/features/modal-context';
-
-import { Illustration } from './icons';
+import { Illustration } from '@packages/assets/icons/Illustration';
```

Comment:
Não to gostando muito desse esquema de import. Acho queque seria bom ter um arquivo `index.ts` na pasta icons pra centralizar os exports.